ovs/src/main/java/org/dcsa/conformance/standards/ovs/party/OvsPublisher.java

```diff
@@ -44,6 +44,11 @@ public OvsPublisher(
         orchestratorAuthHeader);
   }
 
+  private static final boolean RETURN_EMPTY_RESPONSE = false;
+  private static final boolean USE_WRONG_ATTRIBUTE_VALUES = false;
+  private static final boolean USE_WRONG_DATE_TIMES = false;
+  private static final boolean USE_WRONG_RESPONSE_STRUCTURE = false;
```

Comment:
@preetamnpr, I don't understand the idea behind this structure. This is always false in this class, why are we then using it in the logic, to check on it's value? 
And on line 139 and next: you don't need `else` when the previous statement already returns a value. That prevents not necessary indentation and helps readability.